Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.6-alpine
+WORKDIR /usr/src/app
+COPY ./ /usr/src/app
+RUN pip install -e .
+WORKDIR numismatic
+CMD coin listen -f poloniex -a btc collect run
```

Comment:
Very happy to have a minimal Dockerfile.

I think the `CMD` will need to be something more generic though.

Reply:
Yes, I put that command there for a demo. The minimalism is all due to you removing Pandas... Pandas needed some very hectic dependencies in order for it to work...

Reply:
I don't think the Dockerfile is the place to have a demo. I'll submit a commit with a suggestion once I can try it on a Linux box.

Reply:
Yes, I agree. When I get back home, on the 28th, I will change it to work better...